terra-notebooks-playground/README.md

```diff
@@ -1,3 +1,10 @@
+**IMPORTANT NOTE FOR USERS NEW TO JUPYTER NOTEBOOKS:**
+
+* *Setup* notebooks are used to set up computational environments that are language-specific. This means that if a workspace contains both Python and R-based notebooks, and has *Setup* notebooks to run beforehand, users must run the appropriate *Setup* notebook before running subsequent notebooks *in that language*. In other words, if you run the ***R Environment Setup*** notebook in this workspace, and then attempt to run the Python-based notebooks, it will not work. To fix this, you will need to run the ***Python Environment Setup*** notebook within this cluster first.
+
+* Not all warning messages are bad! Often you will run a cell that will return warnings in the output (sometimes these warnings will even be alarmingly color-coded in red or pink). This does not mean that the cell was not successful, and these warnings can often be ignored.
+
+
 **A workspace for trying out Terra functionality as it evolves**.
```

Comment:
Consider moving as least  `**A workspace for trying out Terra functionality as it evolves**.` to the top as the top few lines will be what is display on the 'card view' of the Workspace.

Reply:
Done!